examples/with-thirdparty/.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
```

Comment:
shouldn't the whole .idea folder be in gitignore?

Reply:
Left the defaults based on what Android Studio generates, I can add the `.idea` folder to this though. In general it isnt always needed to ignore all the files just some user specific ones